core/internal/features_test.go

```diff
@@ -118,143 +118,6 @@ func TestIntegration_HttpRequestWithHeaders(t *testing.T) {
 	eth.EventuallyAllCalled(t)
 }
 
-func TestIntegration_FeeBump(t *testing.T) {
```

Comment:
What's the reasoning for removing this test?

Reply:
This test was always broken, because the speed of decrypting keys has gone up it exposed the problem.

I figured it was easier to remove it because it applies only to legacy TXM and we're removing that anyway.

Instead, I fixed the test in the latest commit.